airflow/providers/google/cloud/operators/postgres_to_gcs.py

```diff
@@ -94,7 +94,7 @@ def convert_type(self, value, schema_type):
             return datetime.timedelta(
                 hours=formated_time.tm_hour,
                 minutes=formated_time.tm_min,
-                seconds=formated_time.tm_sec).seconds
+                seconds=formated_time.tm_sec).total_seconds()
```

Comment:
This one is I think wrong.

Reply:
```
In [7]: timedelta(hours=12, minutes=30, seconds=30).seconds                                                                                                                          
Out[7]: 45030

In [8]: timedelta(hours=12, minutes=30, seconds=30).total_seconds()                                                                                                                  
Out[8]: 45030.0
```

(total_seconds returns a float, previously this was an int. I don't think that is intended.)